Resources/config/mongodb.xml

```diff
@@ -56,6 +56,9 @@
     <services>
         <!-- defaults -->
         <service id="doctrine_mongodb.odm.cache" alias="doctrine_mongodb.odm.cache.array" />
+        <service id="Doctrine\Common\Persistence\ObjectManager" alias="doctrine_mongodb.odm.document_manager" public="false" />
```

Comment:
No good - other mappers may also be implementing this interface. Use the `DocumentManager` interface.